### PR TITLE
[CWAG] Fix PipelineD build failure for CWAG

### LINKED ISSUE #6152: [CWAG][Docker][PipelineD] CWAG PipelineD does not build since PR6092
The CWAG PipelineD build has been broken since https://github.com/magma/magma/pull/6092 on master

To reproduce
```
cd magma/cwf/gateway/docker
docker-compose build pipelined
```

Error logs:
```
➜  docker git:(master) ✗ docker-compose build pipelined
WARNING: The ANALYTICS_GRAPHQL_TOKEN variable is not set. Defaulting to a blank string.
WARNING: The ANALYTICS_GRAPHQL_ENDPOINT variable is not set. Defaulting to a blank string.
Building pipelined
[+] Building 2.1s (33/55)
 => [internal] load build definition from Dockerfile                                                                                                                                                                                   0.0s
 => => transferring dockerfile: 5.73kB                                                                                                                                                                                                 0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                      0.0s
 => => transferring context: 35B                                                                                                                                                                                                       0.0s
 => [internal] load metadata for docker.io/library/ubuntu:focal                                                                                                                                                                        0.0s
 => [internal] load build context                                                                                                                                                                                                      0.3s
 => => transferring context: 13.72MB                                                                                                                                                                                                   0.3s
 => CACHED [builder  1/14] FROM docker.io/library/ubuntu:focal                                                                                                                                                                         0.0s
 => CACHED [lte_gateway_python  2/39] RUN apt-get update &&     apt-get install -y apt-utils software-properties-common apt-transport-https                                                                                            0.0s
 => CACHED [lte_gateway_python  3/39] COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub                                                                                                                            0.0s
 => CACHED [lte_gateway_python  4/39] COPY cwf/gateway/deploy/roles/ovs/files/magma-preferences /etc/apt/preferences.d/                                                                                                                0.0s
 => CACHED [lte_gateway_python  5/39] RUN apt-key add /tmp/jfrog.pub &&     apt-add-repository "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" &&     apt-add-repository "deb http://archive.ubunt  0.0s
 => CACHED [lte_gateway_python  6/39] RUN apt-add-repository "ppa:deadsnakes/ppa"                                                                                                                                                      0.0s
 => CACHED [lte_gateway_python  7/39] RUN apt-get -y update && apt-get -y install python3.8                                                                                                                                            0.0s
 => CACHED [lte_gateway_python  8/39] RUN apt-get -y update && apt-get -y install     curl     libc-ares2     libev4     libffi-dev     libjansson4     libjemalloc2     libssl-dev     libsystemd-dev     magma-nghttpx=1.31.1-1      0.0s
 => CACHED [lte_gateway_python  9/39] RUN python3.8 -m pip install --no-cache-dir     Cython     fire     envoy     glob2     ryu     flask     aiodns     pymemoize     wsgiserver     pycryptodome     six     eventlet     h2       0.0s
 => CACHED [lte_gateway_python 10/39] COPY cwf/gateway/deploy/roles/ovs/files/nx_actions.py /usr/local/lib/python3.8/dist-packages/ryu/ofproto/                                                                                        0.0s
 => CACHED [lte_gateway_python 11/39] RUN mkdir -p /var/opt/magma/scripts                                                                                                                                                              0.0s
 => CACHED [lte_gateway_python 12/39] COPY cwf/gateway/deploy/roles/cwag/files/add_uplink_bridge_flows.sh /var/opt/magma/scripts                                                                                                       0.0s
 => CACHED [lte_gateway_python 13/39] RUN git clone --depth 1 --single-branch --branch v2.12.0 https://github.com/openvswitch/ovs.git                                                                                                  0.0s
 => CACHED [lte_gateway_python 14/39] COPY cwf/gateway/deploy/roles/ovs/files/0001-Add-custom-IPDR-fields-for-IPFIX-export.patch /tmp                                                                                                  0.0s
 => CACHED [lte_gateway_python 15/39] COPY cwf/gateway/deploy/roles/ovs/files/0002-ovs-Handle-spaces-in-ovs-arguments.patch /tmp                                                                                                       0.0s
 => CACHED [lte_gateway_python 16/39] COPY cwf/gateway/deploy/roles/ovs/files/0003-Add-pdp_start_epoch-custom-field-to-IPFIX-export.patch /tmp                                                                                         0.0s
 => CACHED [lte_gateway_python 17/39] COPY cwf/gateway/deploy/roles/ovs/files//0004-ovsdb-idlc.in-dict-changes.patch /tmp                                                                                                              0.0s
 => CACHED [lte_gateway_python 18/39] WORKDIR ovs                                                                                                                                                                                      0.0s
 => CACHED [lte_gateway_python 19/39] RUN git apply /tmp/0001-Add-custom-IPDR-fields-for-IPFIX-export.patch                                                                                                                            0.0s
 => CACHED [lte_gateway_python 20/39] RUN git apply /tmp/0002-ovs-Handle-spaces-in-ovs-arguments.patch                                                                                                                                 0.0s
 => CACHED [lte_gateway_python 21/39] RUN git apply /tmp/0003-Add-pdp_start_epoch-custom-field-to-IPFIX-export.patch                                                                                                                   0.0s
 => CACHED [lte_gateway_python 22/39] RUN git apply /tmp/0004-ovsdb-idlc.in-dict-changes.patch                                                                                                                                         0.0s
 => CACHED [lte_gateway_python 23/39] RUN ./boot.sh                                                                                                                                                                                    0.0s
 => CACHED [lte_gateway_python 24/39] RUN ./configure --prefix=/usr --localstatedir=/var --sysconfdir=/etc                                                                                                                             0.0s
 => CACHED [lte_gateway_python 25/39] RUN make                                                                                                                                                                                         0.0s
 => CACHED [lte_gateway_python 26/39] RUN make install                                                                                                                                                                                 0.0s
 => CACHED [lte_gateway_python 27/39] COPY orc8r/gateway/python /tmp/orc8r                                                                                                                                                             0.0s
 => CANCELED [lte_gateway_python 28/39] RUN python3.8 -m pip install --no-cache-dir /tmp/orc8r                                                                                                                                         1.6s
 => ERROR [builder  2/14] RUN apt-get -y update && apt-get -y install curl make virtualenv zip   apt-utils software-properties-common apt-transport-https                                                                              1.6s
------
 > [builder  2/14] RUN apt-get -y update && apt-get -y install curl make virtualenv zip   apt-utils software-properties-common apt-transport-https:
#38 0.442 Get:1 http://security.ubuntu.com/ubuntu focal-security InRelease [109 kB]
#38 0.627 Err:1 http://security.ubuntu.com/ubuntu focal-security InRelease
#38 0.627   At least one invalid signature was encountered.
#38 0.669 Get:2 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
#38 1.230 Err:2 http://archive.ubuntu.com/ubuntu focal InRelease
#38 1.230   At least one invalid signature was encountered.
#38 1.279 Get:3 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
#38 1.365 Err:3 http://archive.ubuntu.com/ubuntu focal-updates InRelease
#38 1.365   At least one invalid signature was encountered.
#38 1.414 Get:4 http://archive.ubuntu.com/ubuntu focal-backports InRelease [101 kB]
#38 1.505 Err:4 http://archive.ubuntu.com/ubuntu focal-backports InRelease
#38 1.505   At least one invalid signature was encountered.
#38 1.509 Reading package lists...
#38 1.520 W: GPG error: http://security.ubuntu.com/ubuntu focal-security InRelease: At least one invalid signature was encountered.
#38 1.520 E: The repository 'http://security.ubuntu.com/ubuntu focal-security InRelease' is not signed.
#38 1.520 W: GPG error: http://archive.ubuntu.com/ubuntu focal InRelease: At least one invalid signature was encountered.
#38 1.520 E: The repository 'http://archive.ubuntu.com/ubuntu focal InRelease' is not signed.
#38 1.520 W: GPG error: http://archive.ubuntu.com/ubuntu focal-updates InRelease: At least one invalid signature was encountered.
#38 1.520 E: The repository 'http://archive.ubuntu.com/ubuntu focal-updates InRelease' is not signed.
#38 1.520 W: GPG error: http://archive.ubuntu.com/ubuntu focal-backports InRelease: At least one invalid signature was encountered.
#38 1.520 E: The repository 'http://archive.ubuntu.com/ubuntu focal-backports InRelease' is not signed.
------
executor failed running [/bin/sh -c apt-get -y update && apt-get -y install curl make virtualenv zip   apt-utils software-properties-common apt-transport-https]: exit code: 100
ERROR: Service 'pipelined' failed to build
➜  docker git:(master) ✗
```

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

## Summary
the CWAG pipelined uses a focal image. https://github.com/magma/magma/issues/6152 split up python dependencies between focal and debian, which broke some python dependencies.

Also modifying the fabfile to only force provision if it is the first time accessing the VM. (was slowing down my testing) + ran ran formatter. (Will address other lints in a separate PR)

I think there's a better way to fix this, but since CWAG/XWF has been broken for ~2 weeks and we want a clean 1.5 release, I'm going to go with this approach unless someone suggests otherwise.

## Test Plan
`docker-compose build pipelined` in both `cwf/gateway/docker` and `xwf/docker` work
running integration tests...
[X] passes locally (DONE 40 tests, 11 skipped in 542.147s)
[ ] passes on circleci
[ ] passes on jenkins

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by themarwhal** [comment]
Force merging to unblock the release process...

**Comment by themarwhal** [comment]
The CWAG integration test passes locally but still fails on CI. I'll investigate on whether there is a separate issue going on :/

File: cwf/gateway/fabfile.py [hunk 1]
```diff
@@ -45,11 +60,13 @@ def list():
...
-               transfer_images=False, destroy_vm=False, no_build=False,
-               tests_to_run="all", skip_unit_tests=False, test_re=None,
-               test_result_xml=None, run_tests=True, count="1"):
+def integ_test(
+    gateway_host=None, test_host=None, trf_host=None,
+    gateway_vm="cwag", gateway_ansible_file="cwag_dev.yml",
+    transfer_images=False, destroy_vm=False, no_build=False,
+    tests_to_run="all", skip_unit_tests=False, test_re=None,
+    test_result_xml=None, run_tests=True, count="1", force_provision=True,
+):
```
Comment (uri200) [inline cwf/gateway/fabfile.py:69]: what about we use `provision_vm` or we change lte_integ_test to `force_provision` for consistency?

Comment (uri200) [reply cwf/gateway/fabfile.py:69]: ops just saw it is merged :D

Comment (themarwhal) [reply cwf/gateway/fabfile.py:69]: ya i can change in a separate PR :)

**Pull request merged**